design/gir/src/main/java/com/automationrockstars/bmo/ContextStoryReporter.java

```diff
@@ -0,0 +1,84 @@
+package com.automationrockstars.bmo;
+
+import java.util.Map;
+
+public class ContextStoryReporter implements StoryReporter {
+
+    @Override
+    public String name() {
+        return "CONTEXT";
+    }
+
+    @Override
+    public void start() {
+        Context.forName(Context.GENERAL);
+    }
+
+    @Override
+    public void finish() {
+        Context.forName(Context.GENERAL).clear();
+    }
+
+    @Override
+    public void beforeStory(String name, String description, String path) {
+
```

Comment:
Whats the story with these empty methods?

Reply:
I need only after... methods to clean context. There is no empty SAbstractStoryReporter I could extend yet.